src/adapter/z-stack/constants/common.ts

```diff
@@ -315,4 +196,89 @@ const COMMON: {
     }
 };
 
-export default COMMON;
\ No newline at end of file
+export enum ZnpCommandStatus {
```

Comment:
Why was this moved out of `COMMON` (in my opinion it makes it inconsistent with the rest).

Reply:
Agree. Using some regexp magic I made them enums and moved everything under the COMMON namespace in `index.ts`. Also did a bit of cleanup in files which use those constants to make code a bit less verbose.